tests/e2e/framework/kubernetes.go

```diff
@@ -645,10 +656,14 @@ func (k *KubeInfo) deepCopy(src map[string][]string) map[string][]string {
 func (k *KubeInfo) deployIstio() error {
 	istioYaml := nonAuthInstallFileNamespace
 	if *multiClusterDir != "" {
-		if *authEnable {
-			istioYaml = mcAuthInstallFileNamespace
+		if *splitHorizon {
```

Comment:
This test will run the same regardless of whether auth is enabled and disabled.  I assume it won't pass in both cases.  It would be nice to throw an error if someone tries to run it with the wrong auth option (assuming that is auth disabled).  I realize the current framework is littered with incompatible options so I wouldn't hold up the review if you don't feel it is worth fixing this one.

Reply:
Fixed in https://github.com/istio/istio/pull/12709/commits/e77f11786cb3145f6d3a9ef3dd336d8396b1d259